README.md

```diff
@@ -15,17 +15,28 @@ This tool runs [ffmpeg](https://www.ffmpeg.org/) commands. To install with brew
 `brew install ffmpeg`
 
 # Usage
+
+## Local
+You can run this tool locally by installing Python and ffmpeg using the dependency above.
 `./clip_extractor.py --source SOURCE_FOLDER --output OUTPUT_FOLDER [--name-pattern NAME_PATTERN] [--encoding-mode ENCODING_MODE] [--verbose]`
 
-`SOURCE_FOLDER` is the folder which will be searched recursively for `.clips` files.
+`SOURCE_FOLDER` (-s) is the folder which will be searched recursively for `.clips` files.
 
-`OUTPUT_FOLDER` is the folder where the clips and manifest file will be written.
+`OUTPUT_FOLDER` (-o) is the folder where the clips and manifest file will be written.
 
 `NAME_PATTERN` is an optional pattern which will be used when creating the clip file names. The tokens `{source}`, `{name}`, `{start_time}` and `{end_time}` will be replaced with the source video file name, the name of the clip, the start time stamp of the clip and the end time stamp of the clip respectively. The default is `{name}.mp4`.
 
-`ENCODING_MODE` specifies how ffmpeg will encode the output clips. Allowed values are `copy-codecs` which will copy the audio and video codecs from the source video, and `re-encode` which will encode the clip as an entirely new video file. `copy-codecs` will be faster but may cause issues in some video players, explanation [here](https://stackoverflow.com/a/18449609). The default mode is `re-encode`.
+`ENCODING_MODE` (-e) specifies how ffmpeg will encode the output clips. Allowed values are `copy-codecs` which will copy the audio and video codecs from the source video, and `re-encode` which will encode the clip as an entirely new video file. `copy-codecs` will be faster but may cause issues in some video players, explanation [here](https://stackoverflow.com/a/18449609). The default mode is `re-encode`.
+
+`--verbose` (-v) is used to show the output of ffmpeg, which is hidden by default.
+
+### Example
+`./clip_extractor.py --source ~/Videos/MySnowboardingHoliday --output ~/Videos/SnowboardingClips`
+
+## Docker
+Alternatively this tool has been adapted slightly to run inside a docker container. This removes the requirement to install Python and ffmpeg locally and can be run inside the container instead. For this to work you need to bind a volume in docker (using the -v) and pairing your local directory to another one mounted in the docker machine. You can then run these through and write files. The only downside to running it this way is that the output need to go inside the main videos folder, although this can be put inside a subfolder if desired.
 
-`--verbose` is used to show the output of ffmpeg, which is hidden by default.
+### Example
+In order to run this in docker you can use the following command as an example:
 
-# Example
-`./clip_extractor.py --source ~/Videos/MySnowboardingHoliday --output ~/Videos/SnowboardingClips`
\ No newline at end of file
+`docker run -v ABSOLUTE_PATH_TO_VIDEOS_LOCALLY:/videos jaymehs/clip-extractor -o /videos/output -s /videos -e copy-codecs -v`
```

Comment:
I don't like seeing a username in the example, plus it would be good to include building the image.
Maybe:
```
docker build . -t clip-extractor
docker run -v ~/Videos/MySnowboardingHoliday:/videos clip-extractor -o /videos/output -s /videos
```
So that it's close to the existing example.

Reply:
Fair enough, the username is there because you can pull a prebuilt image directly from docker hub skipping the build step since it has been done prior.

I'll update the example for this instead though.

Reply:
Pushed an update now 👍